tests/Moq.Tests/ItIsAnyTypeFixture.cs

```diff
@@ -94,6 +94,37 @@ public void Setup_with_It_IsAnyType_default_return_value()
 			Assert.Empty((int[])result);
 		}
 
+		[Fact]
+		public void In_order_to_discover_type_arguments_Callback_accepts_a_single_IInvocation_parameter()
```

Comment:
Should rename: `a_single_IInvocation_parameter` &rarr; `an_InvocationFunc_parameter`.